ramls/migration/migration-job-collection-response.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Get collection of async jobs",
+  "type": "object",
+  "properties": {
+    "jobs": {
+      "description": "List of async jobs records",
+      "javaType": "java.util.List<org.folio.rest.jaxrs.model.AsyncMigrationJob>"
```

Comment:
Curious why it is not a reference to the individual `job` JSON schema?

Reply:
Unfortunately, Rmb creates a separate class for it